dvc/logger.py

```diff
@@ -142,9 +142,22 @@ def handleError(self, record):
         super().handleError(record)
         raise LoggingException(record)
 
+    def emit_pretty_exception(self, exc, verbose: bool = False):
+        return exc.__pretty_exc__(verbose=verbose)
```

Comment:
A better solution would be to just handle it in `dvc.main` and introduce `DvcException.pretty_print()` kinda API, but right now, we just use `logger.exception` everywhere which makes it tricky, will have to get rid of that. For now, we are stuck with this hack.